### PR TITLE
Update release process

### PR DESCRIPTION
**Pull request by k15r**
**Description**

Changes proposed in this pull request:

- address issues found during release of kyma 1.13

**Related issue(s)**

### CODE REVIEW

**Comment by valentinvieriu** [comment]
Is this ready to be merged or is still WIP?

**Comment by k15r** [comment]
it's still wip

File: guidelines/releases-guidelines/02-kyma-release-process.md [hunk 1]
```diff
@@ -12,168 +12,212 @@ A Kyma release includes the following items:
...
+
+Ensure that the `prow/RELEASE_VERSION` file from the `test-infra` repository on a release branch contains the correct version to be created. If you define a release candidate version, a pre-release is created.  
+
+1. Make sure the `RELEASE_VERSION` file includes just a single line, **without the newline character at the end**:  
+
+        ```bash
+        echo -n {RELEASE_VERSION} > prow/RELEASE_VERSION
+        ```
+    
+2. If you had to change the RELEASE_VERSION create a PR to update it on the release branch
```
Comment (bszwarc) [inline guidelines/releases-guidelines/02-kyma-release-process.md:98]:
```suggestion
2. If you had to change the RELEASE_VERSION, create a PR to update it on the release branch.
```

File: guidelines/releases-guidelines/02-kyma-release-process.md [hunk 2]
```diff
@@ -12,168 +12,212 @@ A Kyma release includes the following items:
...
+3. Once this PR is merged you can proceed
```
Comment (bszwarc) [inline guidelines/releases-guidelines/02-kyma-release-process.md:99]:
```suggestion
3. Once this PR is merged you can proceed.
```

File: guidelines/releases-guidelines/02-kyma-release-process.md [hunk 3]
```diff
@@ -12,168 +12,212 @@ A Kyma release includes the following items:
...
+9. Create a new sheet in the [Release Testing](https://docs.google.com/spreadsheets/d/1ty3OciQzgzv0GagTG2Dku9os2AfMupbGNf8QxjHaO88)
+   >**NOTE:** Make sure that you are not signed in with your SAP Google Account
+   1. Open the `Main`
+   2. Click the `generate new sheet` button
+   3. it will ask you for a github personal access token. This token does not need any additional scopes!
+
 
 > **NOTE:** After the Kyma release is complete, proceed with [releasing Kyma CLI](/guidelines/releases-guidelines/03-kyma-cli-release-process.md).
+
+## Post Release Tasks
```
Comment (bszwarc) [inline guidelines/releases-guidelines/02-kyma-release-process.md:221]:
```suggestion
## Post-release tasks
```

File: guidelines/releases-guidelines/02-kyma-release-process.md [hunk 4]
```diff
@@ -12,168 +12,212 @@ A Kyma release includes the following items:
...
    - Instructions on local Kyma installation
    - Instructions on cluster Kyma installation
    - Release notes
 
    For installation instructions, use the links from the previous release and update the version number in URLs. If contributors want you to change something in the instructions, they would address you directly. Contact technical writers for the link to release notes.
 
-11. Create a spreadsheet with all open issues labeled as `test-missing`. Every team assigned to an issue must cover the outstanding test with manual verification on every release candidate. After the test is finished successfully, the responsible team must mark it as completed in the spreadsheet. Every issue identified during testing must be reported. To make the testing easier, provision a publicly available cluster with the release candidate version after you complete all steps listed in this document.
+9. Create a new sheet in the [Release Testing](https://docs.google.com/spreadsheets/d/1ty3OciQzgzv0GagTG2Dku9os2AfMupbGNf8QxjHaO88)
+   >**NOTE:** Make sure that you are not signed in with your SAP Google Account
+   1. Open the `Main`
```
Comment (bszwarc) [inline guidelines/releases-guidelines/02-kyma-release-process.md:214]: the **main** ? 
main view/section? 
also UI elements should be in bold :)

File: guidelines/releases-guidelines/02-kyma-release-process.md [hunk 5]
```diff
@@ -12,168 +12,212 @@ A Kyma release includes the following items:
...
+   2. Click the `generate new sheet` button
```
Comment (bszwarc) [inline guidelines/releases-guidelines/02-kyma-release-process.md:215]:
```suggestion
   2. Click the **generate new sheet** button.
```

File: guidelines/releases-guidelines/02-kyma-release-process.md [hunk 6]
```diff
@@ -12,168 +12,212 @@ A Kyma release includes the following items:
...
+   3. it will ask you for a github personal access token. This token does not need any additional scopes!
```
Comment (bszwarc) [inline guidelines/releases-guidelines/02-kyma-release-process.md:216]:
```suggestion
   3. You will be asked for a GitHub personal access token. This token does not need any additional scopes!
```

File: guidelines/releases-guidelines/03-kyma-cli-release-process.md [hunk 7]
```diff
@@ -33,9 +33,21 @@ A Kyma CLI release consists of:
...
 
-5. Push the tag to trigger a postsubmit job that creates the GitHub release. Validate if the release is available under [releases](https://github.com/kyma-project/cli/releases).
+5. Create the new version
+   1. Push the tag to trigger a postsubmit job that creates the GitHub release. Validate if the release is available under [releases](https://github.com/kyma-project/cli/releases).
+
+      ```bash
+      git push upstream {RELEASE_VERSION}
+      ```
+
+   2. verify the [Prow Status](https://status.build.kyma-project.io/?repo=kyma-project%2Fcli&type=postsubmit) of the matching revision ({RELEASE_VERSION})
```
Comment (bszwarc) [inline guidelines/releases-guidelines/03-kyma-cli-release-process.md:47]:
```suggestion
   2. Verify the [Prow Status](https://status.build.kyma-project.io/?repo=kyma-project%2Fcli&type=postsubmit) of the matching revision ({RELEASE_VERSION}).
```

File: guidelines/releases-guidelines/03-kyma-cli-release-process.md [hunk 8]
```diff
@@ -33,9 +33,21 @@ A Kyma CLI release consists of:
...
+   3. if the post submit job failed: You can retrigger it by removing the tag from upstream and pushing it again
```
Comment (bszwarc) [inline guidelines/releases-guidelines/03-kyma-cli-release-process.md:48]:
```suggestion
   3. If the post submit job failed, you can re-trigger it by removing the tag from upstream and pushing it again.
```

File: guidelines/releases-guidelines/02-kyma-release-process.md [hunk 9]
```diff
@@ -12,168 +12,214 @@ A Kyma release includes the following items:
...
    - Instructions on local Kyma installation
    - Instructions on cluster Kyma installation
    - Release notes
 
    For installation instructions, use the links from the previous release and update the version number in URLs. If contributors want you to change something in the instructions, they would address you directly. Contact technical writers for the link to release notes.
 
-11. Create a spreadsheet with all open issues labeled as `test-missing`. Every team assigned to an issue must cover the outstanding test with manual verification on every release candidate. After the test is finished successfully, the responsible team must mark it as completed in the spreadsheet. Every issue identified during testing must be reported. To make the testing easier, provision a publicly available cluster with the release candidate version after you complete all steps listed in this document.
+9. Create a new sheet in the [Release Testing](https://docs.google.com/spreadsheets/d/1ty3OciQzgzv0GagTG2Dku9os2AfMupbGNf8QxjHaO88)
+   >**NOTE:** Make sure that you are not signed in with your SAP Google Account
+   1. Open the **Main** sheet
```
Comment (bszwarc) [inline guidelines/releases-guidelines/02-kyma-release-process.md:216]:
```suggestion
   1. Open the **Main** sheet.
```

**Pull request merged**